requirements/local.in

```diff
@@ -1,7 +1,7 @@
--c constraints.txt
+-r constraints.txt
```

Comment:
Why this change?

Reply:
> Changed the local requirements to take constraints as normal dependencies (it wasn't installing necessary packages, like django-storages)

IMO the `local.txt` was not built correctly (it was missing requirements)